src/crypto/SecretStorage.js

```diff
@@ -80,38 +75,29 @@ export class SecretStorage extends EventEmitter {
     /**
      * Add a key for encrypting secrets.
      *
-     * @param {string} algorithm the algorithm used by the key.
      * @param {object} opts the options for the algorithm.  The properties used
      *     depend on the algorithm given.
      * @param {string} [keyId] the ID of the key.  If not given, a random
      *     ID will be generated.
      *
      * @return {string} the ID of the key
      */
-    async addKey(algorithm, opts, keyId) {
-        const keyData = {algorithm};
+    async addKey(opts, keyId) {
```

Comment:
Should probably keep the `algorithm` parameter as it may be needed in the future if we add other algorithms.